docs/querying/api.md

```diff
@@ -449,8 +449,8 @@ raw numbers.
 
 The keys `"histogram"` and `"histograms"` only show up if the experimental
 native histograms are present in the response. Their placeholder `<histogram>`
-is explained in detail in its own section below. Any one object will only have
-the `"value"`/`"values"` key or the `"histogram"`/`"histograms"` key, but not
+is explained in detail in its own section below. Any one object could have
+the `"value"`/`"values"` key, the `"histogram"`/`"histograms"` key, or 
 both.
```

Comment:
```suggestion
is explained in detail in its own section below. An instant vector object could
have the `"value"` or the `"histogram"` key, but not both. A range vector object 
could have the `"values"` key, or the `"histograms"` key, or both, but for a given timestamp the sample will only be one of float64 or histogram type.
```

@beorn7 makes sense?

Reply:
How about we move this information under the `range vectors` and `instant vectors` headings?